guides/common/modules/proc_installing-postgresql.adoc

```diff
@@ -10,23 +10,23 @@ include::snip_firewalld.adoc[]
 . To install PostgreSQL, enter the following command:
 +
 ----
-# yum install postgresql-server
+# yum install rh-postgresql12-postgresql-server \
+rh-postgresql12-syspaths \
+rh-postgresql12-postgresql-evr
```

Comment:
It's important to note that the evr package is only needed for Katello. However, the rest of the document is also Katello specific by mentioning Candlepin.